pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
```

Comment:
I think you could use the short option and mention the word "module" in the command description -- for example: "Invoke the setup module to display facts about a group of hosts". I feel that would hit a good balance between convenience and explicitness. But it's just a suggestion; if you prefer it this way, I'm fine with it.